src/PerfView/EventViewer/EventWindow.xaml.cs

```diff
@@ -1075,7 +1075,21 @@ public bool Find(string pat)
             int curPos = SelectionStartIndex();
             var startingNewSearch = false;
 
-            string modifiedPat = FilterQueryUtilities.TryExtractFilterQueryExpression(pat, out FilterQueryExpressionTree tree);
+            FilterQueryExpressionTree tree;
+            try
+            {
+                string modifiedPat = FilterQueryUtilities.TryExtractFilterQueryExpression(pat, out tree);
```

Comment:
I did some additional testing and found an example that didn't work.  If you filter the view to just AllocationTick events, and then use the following expression `[TypeName = System.Char[]]`, I get no results even if they do exist.  I can confirm this with `[TypeName contains System.Char]`.  I'm assuming this is because the type is an array.

Reply:
Yes, I think this is expected behavior with the current implementation. Unfortunately, I have developed this in a way where we are matching only on the first [] and ignoring the rest. I have a couple of ideas as to how this can be fixed (I'll take this on post-merge of this PR if that's fine with you).